tests/unit/Model/Command/UserRecommendationTest.php

```diff
@@ -77,14 +80,14 @@ public function shouldAssembleFilters(): void
         $command = UserRecommendation::create('user-id', 333, 'test-scenario', 1.0, 3600);
 
         // Default filter
-        $this->assertSame('valid_to >= NOW', $command->jsonSerialize()['parameters']['filter']);
+        $this->assertSame('', $command->jsonSerialize()['parameters']['filter']);
 
         // Add custom filters to the default one
         $command->addFilter('foo = bar')
```

Comment:
This test method tests just the legacy regexp filters, right?

I'd thus rename it (`shouldAssembleLegacyRgxFilters()` or something like this) and add another one using some basic MQL filters. Test serve as documentation, and right now, there is no testcase actually using MQL filters.